### PR TITLE
Provide fix for online issues

### PR DESCRIPTION
**Pull request by apeltzer**

### CODE REVIEW

File: nfcore_custom.config [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * name here.
  */
 
-config_base = "${params.custom_config_base}/conf"
+config_base = params.custom_config_base ?: "${params.custom_config_base}/conf" : "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}/conf"
```
Comment (sven1103) [inline nfcore_custom.config]:
```suggestion
default_uri = "https://raw.githubusercontent.com/nf-core/configs/master/conf"
config_base = "${params.custom_config_base ?: default_uri}"
```

**Comment by drpatelh** [comment]
Just means you wont have to evaluate `${params.custom_config_version}` too.

**Comment by apeltzer** [comment]
That should be good yes - however all older pipelines having the current version implemented had these `${params.custom_config_version}` set. The new approach will define `custom_config_base`, so should work fine, too 👍

**Comment by sven1103** [comment]
sorry, this was parallel work to what @drpatelh suggested :) Guess both does the job.

**Pull request merged**